client/components/test-mode-notice/index.tsx

```diff
@@ -31,7 +31,7 @@ interface Props {
 
 const nounToUse = {
 	documents: __( 'document', 'woocommerce-payments' ),
-	deposits: __( 'deposit', 'woocommerce-payments' ),
+	deposits: __( 'payout', 'woocommerce-payments' ),
```

Comment:
Would be good to rename the key here too, avoid confusion.

Note also there is `verbToUse.deposits` below as well.

Reply:
I've been updating the code where I can, so there's less to tidy up (or get confused by!) in future. 

See this PR for example, my game plan was to rename as much as I could safely (i.e. no APIs, database; otherwise rename all variables, components, code, UI).

- https://github.com/Automattic/woocommerce-payments/pull/9554

Reply:
I consciously try to avoid code change as it will lead to larger than necessary PR and increase the risk of oversight. Maybe I'll tune down a bit my paranoid in the future PR for this project.